ui/perfherder/alerts/AlertsViewControls.jsx

```diff
@@ -138,7 +139,8 @@ export default class AlertsViewControls extends React.Component {
             />
           ))}
         {pageNums
-          ? pageNums.length > 1 && (
+          ? pageNums.length > 0 &&
+            count !== 1 && (
```

Comment:
I would try to find a single expression to do this as it is enough.
either `? pageNums[pageNums.length-1] > 1 &&`
or `? page > 1 &&`
but I'd choose `? pageNums[pageNums.length-1] > 1 &&` just to be consistent with the condition at line 117.

Reply:
What if `pageNums.length` is 0? In this case, `pageNums.length-1` will be -1, and `pageNums[-1]` will be `undefined`

Reply:
If `pageNums.length` is 0 then `paheNums` at line `140` will be `False` and the evaluation will end.

Reply:
So you base you current check on something which can change at any time?
I think it is risky and not the way to go.